examples/serial/README.md

```diff
@@ -0,0 +1,18 @@
+# Serial Example
+This serial example is for testing serial connections from the FX2 chip. It is
+uses the same setup as the HDMI2USB firmware, which is:
+
+* Baud rate of 115200
+* Serial device /dev/ttyUSB0 (default on linux)
+* RTS -> PA2
```

Comment:
It isn't, or at least the firmware at no point expects input over serial from the host